packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableResizer.ts

```diff
@@ -137,11 +144,40 @@ function onDragging(
     }
 }
 
-function setResizeDivPosition(context: DragAndDropContext, trigger: HTMLElement) {
+function setResizeDivPosition(
```

Comment:
What if we don't change this part and keep using position:fix, can it still work?

Reply:
Fixed has some issues with scrolling, and changing it would mean reworking the values for left and top again. As it is now, just changing it does not work.

Reply:
Scrolling is a separate issue. We can solve it by reapply the position when scroll. And it can be fixed separately.

For now, there is no complain about the scrolling issue yet so it should be lower priority

Reply:
In this change we just need to solve the zIndex issue.

Scroll issue has been there for long time, so no hurry to fix